WordPress/src/main/java/org/wordpress/android/ui/WPWebViewActivity.java

```diff
@@ -254,6 +254,10 @@ protected void configureWebView() {
             }
             webViewClient = new WPWebViewClient(site, mAccountStore.getAccessToken(), allowedURL);
         } else {
+            if (allowedURL == null || allowedURL.size() == 0) {
```

Comment:
Oh I see! makes sense, thanks for pointing that out, I clearly missed the purpose of `allowedURL` and confused it. Will get back on this later